app/src/main/java/com/example/ptvproject/ui/selecttrainline/SelectTrainLineScreen.kt

```diff
@@ -46,16 +41,12 @@ fun SelectTrainLineScreen(
         val selectTrainLineUiState by selectTrainLineViewModel.uiState.collectAsState()
 
         SelectTrainLineContent(
+            modifier = Modifier.padding(it),
             trainLineUiState = selectTrainLineUiState,
             updateTrainLineScreenState = selectTrainLineViewModel::updateTrainLineScreenState,
-            showAlertDialog = selectTrainLineViewModel::showAlertDialog,
-            modifier = Modifier.padding(it)
+            onTrainLineSelected = onTrainLineSelected,
```

Comment:
great coding practice, very nice clean-up and great changes according to previous discussion!